qiskit_ibm_runtime/session.py

```diff
@@ -179,8 +176,16 @@ def run(
 
         return job
 
+    def cancel(self) -> None:
+        """Cancel all pending jobs in a session."""
+        self._active = False
+        if self._session_id:
+            self._service._api_client.cancel_session(self._session_id)
+
     def close(self) -> None:
```

Comment:
Since this is a behavior change will this trigger a minor version bump when this is released? I'm thinking about how my code explicitly calls `close()` today when my session jobs are done running. Now it won't close the session, but the end result is similar - it should close the session indirectly this way because if there are no more jobs to run when accepting_jobs gets set to false the session will close.

But if I had a case where I wanted to kill a session _before_ jobs run and was relying on `close()` to do that, then I'd need to change to use `cancel()`. And that feels like a minor version bump to me. Assuming people define requirements like `qiskit-ibm-runtime~=0.12` then they could opt into the change when ready.

Reply:
Yes, this will trigger a minor version bump - the next release will be `0.13` (hopefully on monday)